src/Services/Qdm/CqmCalculator.php

```diff
@@ -73,7 +73,7 @@ public function calculateMeasure($patients, Measure $measure, $effectiveDate, $e
             foreach ($patient->dataElements as $dataElement) {
                 // We need to look up OIDs and add the first code concept from the value set.
                 // We do this first in case it's in a dataElement that we need to clone for the calculator below
-                if ($dataElement->negationRationale !== null) {
+                if (isset($dataElement->negationRationale) && $dataElement->negationRationale !== null) {
```

Comment:
what's wrong with !empty($dataElement->negationRationale)?

Reply:
Nothing, that'd work too.  Just did the first thing that jumped to my mind :)

Reply:
Oh well, I just jump at any opportunity to nag!:)